setup.py

```diff
@@ -134,8 +134,6 @@ def req_file(filename, folder="requirements"):
 
 extras_require['slu'] = list(chain([extras_require['slu'], extras_require['asr']]))
 
-tests_requirements = extras_require["test"]
```

Comment:
Why are these two being removed ?

Reply:
There is no need for `tests_requires` if we do not execute `pytest setup.py test`, see deprecation guide: https://github.com/pytest-dev/pytest-runner#deprecation-notice

These requirements still installed when `[all]` is used